### PR TITLE
Test version.py

### PR DESCRIPTION
**Pull request by jacquerie**
I really don't like files with 0% test coverage, even when trivial. Other tests will follow in other PRs.

### CODE REVIEW

File: inspire/version.py [hunk 1]
```diff
@@ -31,11 +27,16 @@
 - dev has to be set in development mode (non-release).
 - revision can be set if you want to override the date coming from git.
 
-See the doctest below.
+See the tests.
 
 (This is taken from Invenio repository)
```
Comment (jacquerie) [inline inspire/version.py]: Maybe I should contribute this upstream?

File: inspire/testsuite/test_version.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            def communicate(self):
+                return ['banana', 'banana']
+
+        popen.return_value = MockCall()
+
+        self.assertEqual('0', git_revision())
+
+    def test_git_revision_from_cache(self):
+        """git_revision fetches from its cache when possible."""
+        del sys.modules['inspire.version']  # Ensure that the cache isn't populated.
```
Comment (jacquerie) [inline inspire/testsuite/test_version.py]: Maybe this should be wrapped, like @dset0x suggests: https://bpaste.net/show/58d2b1a0a48d?

File: inspire/version.py (hunk 1 above)
Comment (jalavik) [inline inspire/version.py]: I do not think this still exists anywhere.. it came from initial 2.0 demo overlay.

File: inspire/testsuite/test_version.py (hunk 2 above)
Comment (jalavik) [inline inspire/testsuite/test_version.py]: Sure.

File: inspire/version.py (hunk 1 above)
Comment (jacquerie) [inline inspire/version.py]: :+1:

File: inspire/testsuite/test_version.py (hunk 2 above)
Comment (jacquerie) [inline inspire/testsuite/test_version.py]: :+1:

**Comment by jacquerie** [comment]
Actually, let's wait for #540 to be merged, then I will rebase on it.

**Comment by jacquerie** [comment]
With #540 merged and this rebased and green I think this is good to go.

**Pull request merged**